src/macros.jl

```diff
@@ -518,6 +518,16 @@ function build_constraint(_error::Function, expr, lb, ub)
     end
 end
 
+function build_constraint(
+        error_func::Function, x::Vector{<:AbstractJuMPScalar}, set::MOI.SOS1)
```

Comment:
`error_func` should be `_error` for consistency.

Reply:
If there is a good argument for `error_func`, we should change everything to `error_func`

Reply:
The argument is ["never begin the name of a local variable with an underscore"](https://www.juliaopt.org/JuMP.jl/latest/style/#Exported-and-non-exported-names-1).

Reply:
Ok, then I suggest making a decision on how to name this argument and stick with it because it will affect many packages (PolyJuMP, SumOfSquares, SetProg, ...). `error_func` looks good to me.

Reply:
Note that for these two methods, it's probably better not to give any name since it is not used